tests/unit/test_attribute_collection.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+
+class TestAttributeCollection(TestCase):
```

Comment:
for new tests, please use [`pytest` fixtures](https://docs.pytest.org/en/latest/fixture.html) instead of `unittest.TestCase`